sources/core/Stride.Core/PlatformFolders.cs

```diff
@@ -186,6 +186,8 @@ private static string GetApplicationBinaryDirectory()
         {
 #if STRIDE_PLATFORM_ANDROID
             return GetApplicationExecutableDirectory();
+#elif STRIDE_PLATFORM_DESKTOP
+            return Path.GetDirectoryName(AppContext.BaseDirectory);
```

Comment:
Is this a correct path if **not** published as single file?

Reply:
I believe so. My tests included both single file and non single file published builds, all launched without issue.